spring-xd-integration-test/src/test/java/org/springframework/xd/integration/test/AbstractIntegrationTest.java

```diff
@@ -265,28 +265,34 @@ public void setupSourceDataFiles(String sourceDir, String fileName, String data)
 	 * @param sourceDir The directory to place the file
 	 * @param fileName The name of the file where the data will be written
 	 * @param data The data to be written to the file
+	 * @result returns true if file was created else false
 	 */
-	public void setupDataFiles(String host, String sourceDir, String fileName, String data) {
+	public boolean setupDataFiles(String host, String sourceDir, String fileName, String data) {
 		Assert.hasText(host, "host must not be empty nor null");
 		Assert.hasText(fileName, "fileName must not be empty nor null");
 		Assert.notNull(sourceDir, "sourceDir must not be null");
 		Assert.notNull(data, "data must not be null");
-
+		boolean result = false;
 		if (xdEnvironment.isOnEc2()) {
-			StreamUtils.createDataFileOnRemote(xdEnvironment.getPrivateKey(), host, sourceDir, fileName, data,
+			result = StreamUtils.createDataFileOnRemote(xdEnvironment.getPrivateKey(), host, sourceDir, fileName, data,
 					WAIT_TIME);
 		}
 		else {
 			try {
+				File dirs = new File(sourceDir);
```

Comment:
This looks like it was pulled in with some other unrelated changes under integration-test project